ticket_metrics.view.lkml

```diff
@@ -164,14 +164,15 @@ view: ticket_metrics {
     sql: ${full_resolution_time_in_days__business} ;;
   }
 
-  #   - dimension: full_resolution_time_in_days__calendar
-  #     type: number
-  #     sql: ${TABLE}.full_resolution_time_in_minutes__calendar / 1440
-  #
-  #   - measure: avg_full_resolution_time_in_days__calendar
-  #     type: avg
-  #     sql: ${full_resolution_time_in_days__calendar}
+  dimension: full_resolution_time_in_days__calendar {
+    type: number
+    sql: ${TABLE}.full_resolution_time_in_minutes__calendar / 1440 ;;
+  }
 
+  measure: avg_full_resolution_time_in_days__calendar{
+    type: average
+    sql: ${full_resolution_time_in_days__calendar};;
```

Comment:
space.

Reply:
Also, can we add a value format?  https://docs.looker.com/reference/field-params/value_format_name

In this case I guess `value_format_name: decimal_2` should be fine.